src/components/job/UpsertJob.jsx

```diff
@@ -436,6 +437,18 @@ function UpsertJobCore({ update }) {
             {...register("description", { required: true })}
           />
         </div>
+        <div className="flex flex-row items-center mb-4">
+          <label className="label flex basis-44" htmlFor="instruction">
+            Instructions of applying for the job
```

Comment:
Can you reword the label to "Job Application Instruction" and below the text field add a text which says "Provide instructions on how to apply for this job - email resume, upload resume to a portal or others".  Also remove the placeholder.

Reply:
Also, please do the same for the Job description and shift details placeholder. Remove them and add the text below the text field as info text